services/datalad/datalad_service/common/git.py

```diff
@@ -40,8 +40,22 @@ def git_tag_tree(repo, tag):
     return repo.get(tag_reference.target).tree_id
 
 
+def git_rename_master_to_main(repo):
+    # Make sure the main branch is used, update if needed
+    master_branch = repo.branches.get('master')
+    if master_branch:
+        main_branch = master_branch.rename('main', True)
```

Comment:
Is there any chance there's a dataset with both `master` and `main`? It looks like `Branch.rename(force=True)` will clobber, but if both are present, are we confident that `master` is authoritative and not `main`? I could imagine a situation where someone with git access pushes a stale `master` branch to a dataset that's had `master` renamed to `main`.

Reply:
Right now there are no main branch datasets and OpenNeuro cannot edit them without these fixes. Maybe instead we block master for new pushes? I could add that here.